query-tuning/index.mdx

```diff
@@ -0,0 +1,80 @@
+
+### Query Tuning Features
+
+* Upload EXPLAIN Plans on Demand: Upload plans to analyze and compare queries
+  in real-time and dive deeper into query performance.
+* Organize Query Plans: Creating a workbook allows you to manage your baseline query and
+  its query variants in one place. This is especially useful when you’re testing different
+  query rewrites, index configurations, or join strategies.
+* Manage and run parameter sets across many query variants, either sourced from query
+  samples or by creating parameter sets manually.
+* Highlight performance differences across variants and parameter sets, allowing you to
+  see at a glance whether a query rewrite or other changes are making things better or worse.
+* View key metrics such as cost estimates, runtime, I/O read time, row estimates, and index scans
+  in a clear overview, helping you form a hypothesis about what to try.
+* Compare Plans: You can run a diff between two query plans, helping you visualize and compare
+  execution plans side-by-side. This feature helps debug whether differences in performance
+  are due to plan shape or environmental factors like I/O time or row estimates. 
+
+## How to enable Query Tuning
+
+### New role and permissions
+
+Query Tuning utilizes a separate user permission, called “Tune Queries”. Only users with
+Admin roles will get this permission assigned automatically. This permission allows
+users to create workbooks and to use all associated Query Tuning features.
+
+Users without the permission are able to view existing workbooks, including parameter
+information, but cannot create new workbooks, variants, or record new plans.
+
+### Set up the collector to run EXPLAIN ANALYZE for you
+
+Query Tuning streamlines the process of gathering query plans through both automatic and
+manual workflows. While you can always upload EXPLAIN ANALYZE plans manually, the collector
+can now run EXPLAIN queries automatically for you, saving time and effort. To use this
+automated feature, ensure you're running collector version 0.64.0 or later and follow our
+[documentation for setting up the collector workflow](collector-workflow).
+
+If you prefer not to use the automatic collection, you can continue using the manual workflow.
+
+Both automatic and manual workflows are available to all users with the Tune Queries permission.
+
+## Limitations
+
+* Workbooks are currently not suited for optimizing queries invoked inside of functions
+* Workbooks are currently not suited for testing different index choices by doing CREATE INDEX
+* Data modification (DML) queries are not supported
+* All query executions are limited to 60 seconds (this limit will be configurable in the future)
+* The Collector Workflow requires the use of a function not owned by superuser
+  * If your database provider does not allow creating separate roles, you cannot use the Collector Workflow
+
+[Learn how to navigate and use workbooks next](how-to-use-workbooks)
```

Comment:
This link is not working well. I think we usually do something like `[](/docs/query-tuning/how-to-use-workbooks)`